inc/deprecated/3.6.php

```diff
@@ -1,9 +1,20 @@
-
 <?php
 // phpcs:ignoreFile
 
 defined( 'ABSPATH' ) || exit;
 
+/**
+ * Require deprecated classes.
+ */
+require WP_ROCKET_DEPRECATED_PATH . 'DeprecatedClassTrait.php';
+require WP_ROCKET_DEPRECATED_PATH . 'Engine/Optimization/QueryString/Remove.php';
+require WP_ROCKET_DEPRECATED_PATH . 'Engine/Optimization/QueryString/RemoveSubscriber.php';
```

Comment:
1. Let's use `require_once` to avoid fatal errors.
2. Use `__DIR__` instead of the constant. Why? The files are in this folder.

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2474/commits/8840398e6c675a49784f63380ad98dfc12bfe571.